### PR TITLE
Readme update and building aarch64

### PR DESCRIPTION
**Pull request by KristianJerpetjon**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -22,33 +24,54 @@ pipeline {
...
+          //agent { label 'ubuntu-18.04' }
+          steps {
+            //checkout scm ?
+            build_conan_package("$SRC", "$USER/$CHAN_LATEST", "$PROFILE_x86_64")
+          }
+        }
       }
     }
+
     stage('Build IncludeOS x86_64') {
```
Comment (mnordsletten) [inline Jenkinsfile]: You can remove this `x86_64` build stage. That is already handled inside the parallel.

File: Jenkinsfile [hunk 2]
```diff
@@ -22,33 +24,54 @@ pipeline {
...
       steps {
         build_conan_package("$SRC", "$USER/$CHAN_LATEST", "$PROFILE_x86_64")
       }
     }
-    stage('Build chainloader x86') {
```
Comment (mnordsletten) [inline Jenkinsfile]: Were we finally able to remove the chainloader from the IncludeOS job completely?

**Comment by staiyeba** [comment]
Updated dependency packages for new profile `gcc-7.3.0-lin​ux-aarch64`.

File: Jenkinsfile [hunk 3]
```diff
@@ -22,38 +24,70 @@ pipeline {
...
-        dir('unittests') {
-          sh script: "conan install $SRC/test -pr $PROFILE_x86_64", label: "Conan install"
-          sh script: ". ./activate.sh; cmake $SRC/test -DCMAKE_BUILD_TYPE=Debug", label: "Cmake"
-          sh script: "make -j $CPUS", label: "Make"
-          sh script: "ctest --schedule-random", label: "Ctest"
+        dir('code_coverage') {
+          sh script: "rm -r $COVERAGE_DIR || :", label: "Setup"
+          sh script: "conan install $SRC/test -pr $PROFILE_coverage -s build_type=Debug", label: "conan install"
+          sh script: ". ./activate.sh && cmake -DCOVERAGE=ON -DCODECOV_HTMLOUTPUTDIR=$COVERAGE_DIR $SRC/test", label: "Cmake"
+          sh script: ". ./activate.sh && make -j $CPUS coverage", label: "Make coverage"
```
Comment (mnordsletten) [inline Jenkinsfile]: Are we not running `ctest` for the unit tests?

File: Jenkinsfile [hunk 4]
```diff
@@ -66,31 +100,16 @@ pipeline {
...
-      }
-      post {
-        success {
-           echo "Code coverage: ${env.COVERAGE_ADDRESS}/${env.JOB_NAME}"
-        }
-      }
-    }
+
     stage('Upload to bintray') {
-      when {
```
Comment (mnordsletten) [inline Jenkinsfile]: Don't change this behaviour. We want it like it is on dev. 
when should come right after "upload to bintray" stage, no when in the latest release stage, and a "buildingTag" in the stable release.

**Pull request merged**